File: Python/1037 - Interval.py
n=float(input())
if n < 0 or n > 100:
    print("Fora de intervalo")
elif (0 <= n <= 25):
    print("Intervalo [0,25]")
elif (25 <= n <= 50):
    print("Intervalo (25,50]")
elif (50 <= n <= 75):
    print("Intervalo (50,75]")
elif (75 <= n <= 100):
    print("Intervalo (75,100]")

File: Java/1037 - Interval.java
import java.util.Scanner;
public class Main

{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		float x = sc.nextFloat();
		if (x<=25 && x>=0) 
		    System.out.println("Intervalo [0,25]");
		else if (x>25 && x<=50)
		    System.out.println("Intervalo (25,50]");
		else if (x>50 && x<=75)
		    System.out.println("Intervalo (50,75]");
		else if (x>75 && x<=100)
		    System.out.println("Intervalo (75,100]");
		else    
		    System.out.println("Fora de intervalo");
	}
}
